src/test/suite/fixtures/recorded/languages/python/ditchCondition.yml

```diff
@@ -0,0 +1,23 @@
+languageId: python
+command:
+  spokenForm: ditch condition
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: condition}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: "[aaa for aaa in bbb if ccc]"
+  selections:
+    - anchor: {line: 0, character: 1}
+      active: {line: 0, character: 1}
+  marks: {}
+finalState:
+  documentContents: "[aaa for aaa in bbb ]"
```

Comment:
Hmm, I was expecting the whitespace to disappear too. It looks like it does in the Rust implementation, wonder what's up here.

Reply:
We generally consider this whitespace cleanup out of scope for scope types, as a) we have plans improve this stuff in the future in a more generic way, and b) an autoformatter can fix this one easily.  I'd be inclined to not bother fixing unless you feel strongly

Reply:
Fair enough. Just struck me as slightly odd; Rust condition not working as intended was the big one.